quill-jdbc/src/main/scala/io/getquill/sources/jdbc/JdbcSource.scala

```diff
@@ -67,25 +65,34 @@ class JdbcSource[D <: SqlIdiom, N <: NamingStrategy](config: JdbcSourceConfig[D,
       }
     }
 
-  def execute(sql: String): Int = {
+  def execute(sql: String, generated: Option[String]): Long = {
```

Comment:
That are a lot of places to change :smile: 

Do you think we should only change the 
`def execute(sql: String) ...` methods

or we should also change all
`def execute[T](sql: String, bindParams ...` methods?